label_sync/labels.yaml

```diff
@@ -660,19 +665,20 @@ repos:
         name: area/devstats
         target: both
         addedBy: label
-      - color: cc99ff
```

Comment:
This doesn't actually delete the label. It just orphans it. You need to use the `deleteAfter` to delete a label.